llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1885,9 +2045,11 @@ DebugAbbrevOffsetsRefAdaptor::encodeOffsets(ArrayRef<size_t> Offsets) {
 }
 
 static void partitionCUDie(
+    MCCASBuilder &Builder,
```

Comment:
is Builder used here?

Reply:
AHHH! Since I did the refactor with the review comments, it is no longer needed, my bad!